src/components/CurrencyInputPanel/index.tsx

```diff
@@ -89,15 +94,29 @@ const FixedContainer = styled.div`
   z-index: 2;
 `
 
-export const Container = styled.div<{ selected: boolean; hideInput: boolean; error?: boolean }>`
+export const Container = styled.div<{ selected: boolean; hideInput: boolean; error?: boolean; $outline?: boolean }>`
   border-radius: 16px;
   background-color: ${({ theme, hideInput }) => (hideInput ? 'transparent' : theme.buttonBlack)};
   padding: ${({ hideInput }) => (hideInput ? 0 : '0.75rem')};
-  border: ${({ error, theme }) => (error ? `1px solid ${theme.red}` : 'none')};
+  ${({ error, theme, $outline }) =>
```

Comment:
```suggestion
  border: 1px solid ${({ error, theme, $outline }) => error ? theme.red : outline ? theme.border : 'transparent'};
```

Reply:
I prefer to keep this since the current solution separate the logic which help easier to modify each and and some more css if needed.

Reply:
just a suggestion, make your own decision